CMakeLists.txt

```diff
@@ -207,8 +216,12 @@ if (ONNX_MLIR_ENABLE_STABLEHLO)
   add_compile_definitions(ONNX_MLIR_ENABLE_STABLEHLO)
 endif()
 
-add_subdirectory(utils)
-add_subdirectory(include)
-add_subdirectory(src)
-add_subdirectory(docs)
-add_subdirectory(test)
+if (ONNX_MLIR_ENABLE_PYRUNTIME_LIT)
```

Comment:
I see this (and above): there are some dir that added on both path. Is it that the order of them is important?

Reply:
Only the `src` is added on both path. I do not think the order of add_subdirectory matters. Just to keep the original add_subdirectory together.